src/frontend/shortlist/src/views/ResetPasswordView.vue

```diff
@@ -55,13 +59,20 @@ export default {
       return this.form.newPassword == this.form.confirmPassword;
     },
     submitPWResetForm() {
-      this.$emit("appPasswordReset", {
-        email: this.form.email,
-        currentPassword: this.form.currentPassword,
-        newPassword: this.form.newPassword,
-        confirmPassword: this.form.confirmPassword,
-      });
+      axios
+        .patch("https://api.shortlist.nyc/auth/password-reset-complete", {
+          password: this.form.newPassword,
+          token: this.$route.query.token,
+          uidb64: this.$route.query.uidb64,
+        })
+        .then(function (response) {
+          console.log(response);
+        })
+        .catch(function (error) {
+          console.log(error);
+        });
       console.log(this.form.email);
+      alert("Your password has been reset");
```

Comment:
Should only alert on success. If axios call fails, alert that the password was not reset